packages/anasazi/tpetra/example/TraceMinDavidson/TraceMinDavidsonSpecTransEx.cpp

```diff
@@ -132,6 +127,7 @@ main (int argc, char *argv[])
   //
   // Read the matrices from a file
   //
+  RCP<Tpetra::Map<>::node_type> node; // can be null
   RCP<const CrsMatrix> K = Tpetra::MatrixMarket::Reader<CrsMatrix>::readSparseFile(filenameA, comm, node);
```

Comment:
@ndellingwood Oops, missed that one ;-) .  It's OK to remove it, but not wrong to leave it in there.  It's OK to be null.  Some other interfaces (e.g., the Harwell-Boeing file reader that some tests use) need a `node` argument, but it can be null.